docs/source/data/subscriptions.mdx

```diff
@@ -95,6 +95,13 @@ In order to run both an Express app and a separate subscription server, we're go
        // This `server` is the instance returned from `new ApolloServer`.
        path: server.graphqlPath,
     });
+
+    // Shut down in the case of interrupt and termination signals
+    // We expect to handle this more cleanly in the future. See (#5074)[https://github.com/apollographql/apollo-server/issues/5074] for reference.
+    ['SIGINT', 'SIGTERM'].forEach(signal => {
+      process.on(signal, () => subscriptionServer.close();
```

Comment:
You probably copied this from something I wrote, but syntax looks wrong here (missing close paren)